### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by xoac** [dswd/zvault#14]
Hi this project looks awesome but unmaintained. Is zVault stable enough to be used as backup solution? Maybe it would be worth to add some info to README.md since last build has fail status.

**Comment by dswd** [comment]
Hi, the answer is complicated.

**Stability**
I used zVault over a year and had no major problems with it.

**Project status**
TL;DR: zvault1 is dead, long live zvault2

I am currently doing a fundamental redesign of zvault, to overcome some design limitations. This took a lot of time and is about 80% done. All the hard problems have been solved and the tool just needs a CLI UI. Due to my limited time, I can't say when it will be done. Here are some key points:
* zvault2 will be incompatible with zvault1, no conversion path planned (except export/import of tarballs)
* zvault2 will allow for remote storage that is not mounted like S3
* zvault2 will support parity bundles for increased robustness on lost or broken remote bundles

**Comment by xoac** [comment]
Awesome! Waiting for it!
